### ISSUE TITLE
Can i use this module in iOS project?

### ISSUE THREAD

**Issue by Karthus1110** [jrosebr1/imutils#91]
I want to use this module to call openCV function in my ios application?  I don't sure if this can use in app

**Comment by jrosebr1** [comment]
No, `imutils` is for Python only. You could reimplement a specific function you want to use in a different language, but you cannot call a Python function from your iOS app.
